pkg/datastore/test/caveat.go

```diff
@@ -27,7 +27,7 @@ import (
 func CaveatNotFoundTest(t *testing.T, tester DatastoreTester) {
 	require := require.New(t)
 
-	ds, err := tester.New(0, veryLargeGCWindow, 1)
+	ds, err := tester.New(0, veryLargeGCInterval, veryLargeGCWindow, 1)
```

Comment:
not a big deal, but it doesn't really matter what the gcinterval is here, since GC won't remove anything inside the window